ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml

```diff
@@ -34,6 +34,11 @@ tests:
   steps:
     cluster_profile: aws
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: installer-rehearse-aws-ipi-proxy
+  interval: 960h
+  steps:
+    cluster_profile: aws
+    workflow: cucushift-installer-rehearse-aws-ipi-proxy
```

Comment:
You can directly change the line 36 line to validate the new workflow so we do not have to introduce a new test grid.